instrumentation/thrift-0.14.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/thrift/AbstractProtocolWrapper.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.thrift;
+
+import org.apache.thrift.protocol.TProtocol;
+import org.apache.thrift.protocol.TProtocolDecorator;
+
+public abstract class AbstractProtocolWrapper extends TProtocolDecorator {
+  public static final String OIJ_THRIFT_FIELD_NAME = "OIJ_FIELD_NAME";
+  public static final short OIJ_THRIFT_FIELD_ID = 3333;
```

Comment:
`OIJ_THRIFT_FIELD_ID` is a weird name , what does `OIJ` stand for? I assume that this is the id for the field that you use for context propagation? If so it would be nice to have a comment. Also would it make sense to use a larger number than `3333`? Would that reduce the risk of colliding with an existing field? Would it make sense to make this configurable?

Reply:
Or does it come from https://github.com/opentracing-contrib/java-thrift/blob/2fdf7c55309150fb9e3b06ebf773e37f80968d1b/src/main/java/io/opentracing/thrift/SpanProtocol.java#L52 ?

Reply:
Yes. Make it configurable with a default value
May make sense